test-loop-tests/src/tests/catching_up.rs

```diff
@@ -375,27 +371,12 @@ fn ultra_slow_test_catchup_sanity_blocks_produced() {
 }
 
 #[test]
-fn ultra_slow_test_all_chunks_accepted_1000() {
-    test_all_chunks_accepted_common(1000, 3000, 7)
-}
-
-#[test]
-fn ultra_slow_test_all_chunks_accepted_1000_slow() {
-    test_all_chunks_accepted_common(1000, 6000, 7)
-}
-
-#[test]
-fn ultra_slow_test_all_chunks_accepted_1000_rare_epoch_changing() {
-    test_all_chunks_accepted_common(1000, 1500, 100)
-}
-
-fn test_all_chunks_accepted_common(
-    last_height: BlockHeight,
-    block_prod_time: u64,
-    epoch_length: BlockHeightDelta,
-) {
+fn slow_test_all_chunks_accepted() {
```

Comment:
How long does it take to run now? Even if it falls within the slow test timeout it may still be good to keep it in nayduck if it slows down CI.

Reply:
It's about a minute long, 70 sec or so on my system. nextest runs the tests in parallel and we have a bunch of tests like resharding that are about the same length.